src/tlo/methods/chronicsyndrome.py

```diff
@@ -214,11 +214,11 @@ def initialise_simulation(self, sim):
         self.sim.schedule_event(outreach_event, self.sim.date + DateOffset(months=6))
 
         # Schedule the occurance of a population wide change in risk that goes through the health system:
```

Comment:
```suggestion
        # Schedule the occurrence of a population wide change in risk:
```

Possibly remove the reference to health system in comment as no longer a HSI event?